docs/packages/package_main.rst

```diff
@@ -0,0 +1,65 @@
+.. _Mesa-Packages: 
+
+Mesa Packages 
+==============
+
+*Supporting MESA Agent Based Modeling with Crowd Sourced Packages*  
```

Comment:
I think we should remove this quote. This is because it is more general and gets away from the content.